File: src/server/nodejs/ErpDomain/CustomerModule/CustomerAggregate/Customer.ts
module DevxSample.ErpDomain.CustomerModule.CustomerAggregate {
    /**
     * Descripts a Customer entity.
     */
    export class Customer {
        /**
         * @param firstname The first name of the customer.
         */
        constructor(public firstName: string)
        {
            
        }
        
        public customerPkId: number;
    }
}
    
var customer = new DevxSample.ErpDomain.CustomerModule.CustomerAggregate.Customer('Harry')
console.log(customer.firstName);

File: src/server/dotnetcore/ErpDomain/CustomerModule/CustomerAggregate/Customer.cs
namespace DevxSample.ErpDomain.CustomerModule.CustomerAggregate
{
    public class Customer
    {
        public Customer(string firstName) {
            this.FirstName = firstName;  
        }
        
        public int CustomerPkId { get; set; }
        public string FirstName { get; set; }
    }
}